templates/wheretofindme/user_detail.html

```diff
@@ -1,4 +1,14 @@
 {% extends "base.html" %}
+{% load static %}
+
+{% block "opengraph" %}
```

Comment:
👍 This is cool as heck.

It occurs to me looking at this that adding a slack/twitter/etc preview card might also be a good idea, so that this does something useful if you copy-paste your link into Discord.

Reply:
I know Slack at least pulls from OG tags, so that's why I used them. I think Twitter does the same?